src/assets/fabricator/styles/components/layout.css

```diff
@@ -0,0 +1,67 @@
+.f-Layout {
+  height: 100%;
+}
+
+.f-Layout.f-is-active {
```

Comment:
> So utility classes without namespace could be dangerous, right?

I saw this as a _state_ class. State classes should only ever be defined in combination with another class, so this collision shouldn't occur as long as that _other_ class is prefixed.

```css
.f-SomeClass.is-active { }

.f-SomeOtherClass.is-active { }
```